docs/learningLab/introToLit/acc.md

```diff
@@ -0,0 +1,88 @@
+
+---
+
+## Objectives
+At completion of this reading you should be able to:
+
+- Describe what access control conditions are useful for.
+- List common types of applications that can be developed with access control.
+
+---
+
+## Example Conditions
+
+- User is a member of a DAO
+- User holds an NFT in a collection
+- User holds at least 0.1 ETH
+- The result of any smart contract function call
+- User owns a specific wallet address
+- Using boolean operations (AND + OR) for any of the above
+
+### Supported Logic
+
+- Supports many EVM chains and Solana. Full list [here](https://developer.litprotocol.com/support/supportedChains).
+- Supports many standard contracts, with plans to support any RPC call soon. If you need to interact with a contract that we don't support yet, ask us, and we will implement it.
+- *Boolean conditions*: "And" or "Or" are currently supported.
+- *Updateable conditions*: Only the creator can update the condition.
+- *Permanent conditions*: When a condition is stored as permanent, it becomes impossible to update it, forever.
+
+## Use Cases
+
+With access control, you can introduce private and permissioned data to the open web using decentralized encryption. Use Lit to provision decryption "keys" to users based on on-chain conditions, such as ownership over a specific NFT or membership within a DAO.
+
+Some examples of applications that can (and have) been built harnessing this functionality:
+
+**Web3 Apps with Private Data**
+
+Introduce privacy to decentralized applications. Use Lit to securely store private and permissioned data on the open web. To get started with an integration, use our JavaScript [SDK](https://developer.litprotocol.com/SDK/intro).
+
+Examples:
+
+- [Orbis Club](https://orbis.club/): Enabling “friends-only” content and encrypted messaging.
+- [Lens](https://docs.lens.xyz/docs/gated): Permissioned data for web3 social.
+- [Streamr](https://blog.streamr.network/streamr-integrates-lit-protocol/): Private data marketplaces.
+- [Gather](https://www.gather.town/): Powering private virtual spaces for your team.
+
+**Add Token Gating to Web2 Apps**
+
+Use on-chain credentials to gate access to your favorite web2 applications and data.
+
+- [Zoom](https://litgateway.com/apps/zoom): Token-gated Zoom calls.
+- [Google Drive](https://litgateway.com/apps/google-drive): Add access control requirements to your Google Drive files.
+- [WordPress](https://litgateway.com/apps/wordpress): Gate access to WordPress sites and pages.
+
+---
+
+## Basic Example
```

Comment:
i like the high-level overview and conceptual learning approach at the start of "intro to lit". the code example and sdk reference may feel abrupt here. we could include a callout to using the sdks at the very end of the section - perhaps as a "next steps" section in the PoK page

Reply:
or to match the next "pkp and lit actions" page, nest "Basic Examples" and "Learn More" under "Working with Access Control"